setup.py

```diff
@@ -43,16 +43,16 @@ class Clean(clean):
 
     def run(self):
         super().run()
+        self.clean_optional_plugins()
```

Comment:
Hi ana, sorry for the noise, it didn't worked. So maybe we just need to remove the directory:

```diff
diff --git a/setup.py b/setup.py
index 537238fd..8be3ba8f 100755
--- a/setup.py
+++ b/setup.py
@@ -46,7 +46,7 @@ class Clean(clean):
         call(('rm -rf MANIFEST BUILD BUILDROOT SPECS RPMS SRPMS SOURCES',
               'PYPI_UPLOAD ./build ./dist'), shell=True)
         call('rm -rf ./man/avocado.1 ./docs/build', shell=True)
-        call('rm -rf /var/tmp/avocado* /tmp/avaocado*', shell=True)
+        call('rm -rf /var/tmp/avocado/ /tmp/avocado/', shell=True)
         call('find . -name "*.egg-info" -exec rm -rv {} +', shell=True)
         call('find . -name "*.pyc" -exec rm -rv {} +', shell=True)
         call('find . -name __pycache__ -type d -exec rm -rv {} +', shell=True)
```